Assets/FishFactory/Scripts/FishSortingButton.cs

```diff
@@ -18,28 +18,48 @@ public enum FishTier
     // list of sorted fish
     public List<GameObject> sortedFish = new List<GameObject>();
 
+    // ------------------ Private Variables ------------------
+
+    private Material tier1;
+    private Material tier2;
+    private Material tier3;
+    private GameObject ColorPannel;
+
+    // ------------------ Unity Functions ------------------
+
+    void Awake()
+    {
+        // Load materials
+        tier1 = Resources.Load<Material>("Materials/QAPannel/Tier1");
+        tier2 = Resources.Load<Material>("Materials/QAPannel/Tier2");
+        tier3 = Resources.Load<Material>("Materials/QAPannel/Tier3");
+
+        // Sets the instance of the FishSortingButton to this object if it does not already exist
+        if (fishSortingButton == null)
+        {
+            fishSortingButton = this;
+        }
+        // Find first instance of ColorBox, if there are more than one sorting machine that the player should be able to use this will need to be changed
```

Comment:
What does this mean? I don't understand

Reply:
.Find() gets only one game object, if there are more than 1 game object with the same name then the second one will not change color. This is preferable in our case but might be a problem if the user needs to interact with multiple different sorting machines.